backend/app/core/validators/gender_assumption_bias.py

```
@@ -0,0 +1,78 @@
+            if category == BiasCategories.All:
+                temp = df
+            else:
+                temp = df[df['type'] == category]
+
+            rows = temp.to_dict(orient="records")
+            for row in rows:
+                gender_bias_list.append({
+                    "word": row["word"],
+                    neutral_term_col: row[neutral_term_col]
+                })
+        return gender_bias_list
```

Ruff (0.14.8)

No newline at end of file

Add trailing newline

(W292)